src/testUtils/spectron.js

```diff
@@ -0,0 +1,113 @@
+import { toMatchImageSnapshot } from "jest-image-snapshot";
+import { Application } from "spectron";
+import electron from "electron";
+import path from "path";
+import sharp from "sharp";
+
+expect.extend({ toMatchImageSnapshot });
+
+process.env.REACT_APP_RUNNING_TESTS = true;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
+
+let app;
+
+export async function compareScreenshot() {
+  await waitForReady();
+  const screenshot = await app.client.saveScreenshot();
+  const windowSize = await app.browserWindow.getSize();
+  const resizedScreenshot = await sharp(screenshot)
+    .resize(windowSize[0], windowSize[1])
+    .toBuffer();
+  expect(resizedScreenshot).toMatchImageSnapshot();
```

Comment:
Due to a bug on the webdriverio API used by spectron the screenshot taken is not taken into account the device pixel ratio. This means, that an screenshot taken in a MacBook Pro will be twice bigger than the one taken in a MacBook Air.